public/app/plugins/datasource/tempo/dataquery.gen.ts

```diff
@@ -12,10 +12,6 @@ import * as common from '@grafana/schema';
 
 export interface TempoQuery extends common.DataQuery {
   filters: Array<TraceqlFilter>;
-  /**
-   * Filters that are used to query the metrics summary
-   */
-  groupBy?: Array<TraceqlFilter>;
```

Comment:
What's going to happen with existing saved queries (e.g. in Dashboards, saved URLs)? Would it just silently return non-aggregated results? If so, would it make sense to return error if a query contains `groupBy` to force user to check/edit the query?

Reply:
> (...) to create a new panel.

There are no panels in Explore (in case someone was coming from an URL to Explore). 

Maybe we could also be more explicit and say - "Please remove the current query and create a new one." as this is the only way out of the error (changes to the query won't do anything).